### ISSUE TITLE
diffie-hellman: t.Fatal() instead of t.Fatalf()

### ISSUE THREAD

**Issue by ivenk** [exercism/go#1217]
The TestPrivateKey function in the diffie-hellman-test.go file calls t.Fatal() where I believe it should rather call t.Fatalf().

Line : 100 (diffie-hellman-test.go)
```
        if len(mb) < 100 {
                t.Fatal("For prime %d duplicate keys generated.  "+
                        "Want unique keys.", biggerTest.p)
	}
```

The parameters parsed to the method indicate that the advanced formatting of t.Fatalf() is desired. Additionally every other case uses the t.Fatalf() function as well.

**Comment by kytrinyx** [comment]
Yeah, I think you're right.

**Comment by bitfield** [comment]
This is fixed in 0abf43cf9047b4805529fe9381ad55badddf4942.

**Comment by ivenk** [comment]
Okay great !
